test/foundry/new/helpers/FuzzGenerators.sol

```diff
@@ -327,36 +331,47 @@ library AdvancedOrdersSpaceGenerator {
         AdvancedOrder[] memory orders,
         AdvancedOrdersSpace memory space,
         FuzzGeneratorContext memory context
-    ) internal pure {
+    ) internal {
         for (uint256 i = 0; i < orders.length; ++i) {
             _adjustUnavailable(
-                orders[i].parameters,
+                orders[i],
                 space.orders[i].unavailableReason,
                 context
             );
         }
     }
 
     function _adjustUnavailable(
-        OrderParameters memory order,
+        AdvancedOrder memory order,
         UnavailableReason reason,
         FuzzGeneratorContext memory context
-    ) internal pure {
+    ) internal {
+        OrderParameters memory parameters = order.parameters;
         // UnavailableReason.AVAILABLE => take no action
         if (reason == UnavailableReason.EXPIRED) {
-            order = order.withGeneratedTime(
+            parameters = parameters.withGeneratedTime(
                 Time(context.randEnum(3, 4)),
                 context
             );
         } else if (reason == UnavailableReason.STARTS_IN_FUTURE) {
-            order = order.withGeneratedTime(
+            parameters = parameters.withGeneratedTime(
                 Time.STARTS_IN_FUTURE,
                 context
             );
+        } else if (reason == UnavailableReason.CANCELLED) {
+            order.inscribeOrderStatusCanceled(true, context.seaport);
+        } else if (reason == UnavailableReason.ALREADY_FULFILLED) {
+            order.inscribeOrderStatusNumeratorAndDenominator(
```

Comment:
Shouldn’t this inscribe 1 + 1? We want to update the state to reflect that it’s fully filled, not that it’s already filled with the numerator / denominator we’re attempting to fill?